src/parser/jobs/vpr/modules/DISPLAY_ORDER.ts

```diff
@@ -1,9 +1,11 @@
 export default {
 	BUFFS: 1,
-	NOXIOUS_GNASH: 2,
-	SEPRENTSTAIL: 3,
-	POSITIONALS: 4,
-	COOLDOWNS: 5,
-	SNILK: 6,
-	REAWAKEN: 7,
+	HONED_PROCS: 2,
+	FINISHER_PROCS: 3,
+	VENOM_PROCS: 4,
+	SEPRENTS_TAIL: 5,
```

Comment:
```suggestion
	SERPENTS_TAIL: 5,
```

Update to match module name while we're making other changes here?